### ISSUE TITLE
Errors in tensorflow while training the model

### ISSUE THREAD

**Issue by hariteja97** [ParikhKadam/bidaf-keras#9]
Whenever I try to train the model.. I get this error and I don't know how to rectify this error.
![IMG20190522140118](https://user-images.githubusercontent.com/29245616/58159493-440cad00-7c9a-11e9-8c0b-38ad3e720303.jpg)

This is the error I get.. 

Thanks in advance.

**Comment by ParikhKadam** [comment]
Which version of tensorflow are you using?

**Comment by hariteja97** [comment]
I am using tensorflow 1.13.1 and Keras as 2.2.4.. and it's python 3.6.8
version..

**Comment by ParikhKadam** [comment]
Try using tensorflow 1.12.0 and report is the error persists. 

I am going to update this project to support tensorflow 2.0 and use tf.keras instead of keras as soon as a stable version to tensorflow 2.0 releases..

**Comment by ParikhKadam** [comment]
@hariteja97 Did this solve the issue?

**Comment by hariteja97** [comment]
@kadamparikh. Thanks a lot it helped me a lot ..I have just re installed
1.12.0 version tensorflow which solved my issue .but I was trying the same
code with theano backend but it's not working .

 It would be great if we can implement this in theano also


 One more thing.

1. How to make the model much more accurate like there are some exceptions
with this code like few answers are not developed properly.. to make it
more precise

 Can u explain how to use BERT model and or any other model for making it
more accurate

**Comment by ParikhKadam** [comment]
@hariteja97 The model doesn't work well with Theano is a problem with Keras. You should try opening an issue at Keras Repo with the errors you get. 

Improving model's accuracy needs training. The models I provided aren't actually completely trained. And I don't have resources for further training. Try that yourself..

Thank you..
